std/typecons.d

```diff
@@ -2969,7 +2969,11 @@ Returns:
     {
         enum message = "Called `get' on null Nullable!" ~ T.stringof ~ ".";
         assert(!isNull, message);
-        return _value.payload;
+        inout(T)* get2() inout @trusted pure nothrow
```

Comment:
Except for `@trusted`, everything else should be inferred, no?

Reply:
inout is necessary I think because I got https://github.com/dlang/phobos/pull/7047#issuecomment-496536998 before with one of the previous tries. the other attributes, not sure.

Reply:
Even `inout` probably isn't:

```d
auto get2() @trusted {
    return &_value.payload;
}
```

Reply:
It is, same error.